project2/style.css

```
@@ -1,3 +1,62 @@
+.background {
+    width: 640px;
+    height: 1100px;
+    background-color: #979797;
+
+}
+
+button {
+    position: relative;
+    left: 250px;
+    bottom: 20px;
+    font-size: 15px;
+    background-color: #bb8c4e;
+}
```

See note in the HTML about handling all caps via CSS vs the markup:
```suggestion
  text-transform: uppercase;
}
```